libraries/contact.rb

```diff
@@ -102,6 +102,27 @@ def push(obj)
       end
     end
 
+    # rubocop:disable MethodLength
+    def pop(obj)
+      return if obj == self
+      case obj
+      when Nagios::Contactgroup
+        if @contactgroups.keys?(obj.to_s)
+          pop_object(obj, @contactgroups)
+          pop(self, obj)
```

Comment:
This method is never called, because if it would be called, the following error should appear:
```
 when Nagios::Contactgroup
    if @contact_groups.keys?(obj.to_s)
      pop_object(obj, @contact_groups)
      pop(self, obj)
    end
  when Nagios::Command
    @check_command = nil if @check_command == obj
  when Nagios::Timeperiod
    @check_period = nil if @check_command == obj
    @notification_period = nil if @check_command == obj
  when Nagios::CustomOption
    if @custom_options.keys?(obj.to_s)
      pop_object(obj, @custom_options)
[17] pry(#<Nagios::Service>)> pop(self, obj)
ArgumentError: wrong number of arguments (2 for 1)
from /tmp/kitchen/cache/cookbooks/nagios/libraries/service.rb:202:in `pop'
```